src/Storages/MergeTree/ReplicatedMergeTreeCleanupThread.cpp

```diff
@@ -72,6 +72,7 @@ void ReplicatedMergeTreeCleanupThread::iterate()
         clearOldLogs();
         clearOldBlocks();
         clearOldMutations();
+        storage.clearEmptyParts();
```

Comment:
What if multiple replicas will assign `DROP_RANGE` for the same parts almost at the same time. Will it work well? Maybe we will have some exceptions in logs?